components/NavBar/NavBar.tsx

```diff
@@ -125,7 +125,7 @@ export const NavBar = () => {
         <HeaderContentContainer>
           <LogoContainer>
             <Link href="/">
-              <Button as="a" variant="ghost" color="secondary">
```

Comment:
what prompted this change?

Reply:
It was earlier before  we updated the approach to use next link component rather than ours  . Updated the PR with link component from next now so this change not required.